src/main/java/name/valery1707/kaitai/KaitaiMojo.java

```diff
@@ -128,7 +123,15 @@
 	/**
 	 * Executes the plugin, to read the given source and behavioural properties and generate POJOs.
 	 */
-	public void execute() throws MojoExecutionException, MojoFailureException {
+	public void execute() throws MojoExecutionException {
+		try {
+			executeInt();
+		} catch (KaitaiException e) {
+			throw new MojoExecutionException(e.getMessage(), e.getCause());
+		}
+	}
+
+	private void executeInt() throws KaitaiException {
```

Comment:
\[Checkstyle\] WARNING: 'method def modifier' has incorrect indentation level 8, expected level should be 2\.